changelogs/fragments/1933-zos_blockinfile-false_negative_double_quotes.yml

```diff
@@ -0,0 +1,4 @@
+bugfixes:
```

Comment:
I would change this fragment to a ``trivial`` one, since this change is not really affecting the current user experience (you're changing things behind the scenes, so to speak).

Reply:
I agree with Ivan's suggestion, since behavior is not going to change for the user we should set it as trivial. That doesn't make it less impactful though, you've done changes that will improve the codebase maintainability.